src/js/CommentPopup.js

```diff
@@ -0,0 +1,59 @@
+const url1 = 'https://api.tvmaze.com/shows';
+const fetchMovieData = async (movieId) => {
+  const response = await fetch(`${url1}/${movieId}`)
+    .then((res) => res.json())
+    .then((data) => data)
+    .then((error) => error);
+  return response;
```

Comment:
- [ ] I suggest trying to refactor this function and using the one that already exist in api.js

Reply:
thanks